modules/dropdown/index.ts

```diff
@@ -7,4 +7,12 @@ export {
 export {
   default as MultipleDropdown,
   IMultipleProps as IMultipleDropdownProps
-} from "./MultipleDropdown";
+} from "./BaseMultipleDropdown";
```

Comment:
I also forgot to add a comment asking if we could have one of the Dropdowns be named `MultipleDropdown` without any prefix. The `FilterMultipleDropdown` would be the variant as I see it.
I believe it is more intuitive to have `MultipleDropdown` instead of `SimpleMultipleDropdown`!
What do you think? 💭